week2/week2-homework

```diff
@@ -0,0 +1,117 @@
+# Week-2 homework
+# 1. Write fizz_buzz function
+from pywin.mfc.object import Object
+
+
+def fizz_buzz(number):
+    if number % 3 == 0 and number % 5 == 0:
+        return "FizzBuzz"
+    elif number % 3 == 0:
+        return 'Fizz'
+    elif number % 5 == 0:
+        return 'Buzz'
+    else:
+        return number
+
+
+print(fizz_buzz(12))
+
+# 2. write a try and except block to avoid IndexError
+lst = [5, 10, 20]
+try:
+    index = int(input('inter the index of an item in the list: > '))
+    element = lst[index]
+    print(element)
+except ValueError:
+    print('Index should be Integer number')
+except IndexError:
+    print('list index out of range')
+
+
+# 3. Create a class of Jet inventory with two arguments
+
+
+class Jet:
+    def __init__(self, name, country):
+        self.name = name
+        self.country = country
+        print(f'{name} {country}')
+
+
+jet1 = Jet('helicopter', 'Denmark')
+jet2 = Jet('Boing', 'USA')
+
+# 5. Write a Python script to check whether a given key already exists in a dictionary.
+customers = {'name': 'Jonis', 'phone': '123-345-8',
+             'address': 'CPH'}
+
+
+def check_key(key):
+    return key in customers
+
+
+print(check_key('phone'))
+
+
+# 6. Odd or Even
+
+
+def show_numbers(limit):
+    for num in range(0, limit + 1):
+        if num % 2 == 0:
+            print(f'{num}  EVEN')
+        else:
+            print(f'{num}  ODD')
```

Comment:
Here you could do even better by separating, printing and logic. Do you know what I mean by that?